src/app/components/challenge/challengephases/phasecard/phasecard.component.html

```diff
@@ -1,27 +1,38 @@
-<div class="phase-container">
-	<div class="phase-title">
-		{{phase['name']}}
-	</div>
-	<div class="phase-section">
-		<div class="start">
-			<span class="phase-heading">Starts on</span> <br>
-			<span>{{startDate}}</span>
-		</div>
-		<div class="end">
-		    <span class="phase-heading">Ends on</span><br>
-			<span>{{endDate}}</span>
-		</div>
-	</div>
-	<div class="phase-description" [innerHTML]="phase['description']">
-	</div>
-	<div class="phase-section">
-		<div class="start">
-			<span class="phase-heading">Max number of submissions/day</span> <br>
-			<span>{{phase['max_submissions_per_day']}}</span>
+<div class="bottom-card-container ev-card-panel ev-md-container phase-card">
+	<div class="row row-lr-margin">
+		<div class="col-md-11 col-sm-11 col-lr-pad">
+			<div class="row row-lr-margin">
+				<div class="col-sm-12 col-lr-pad">
+					<span class="fs-18 text-med-black">
+						<strong class="fw-semibold">{{phase['name']}}</strong>
+					</span>
+				</div>
+			</div>
+			<div class="row row-lr-margin">
+				<div class="col-sm-12 col-lr-pad">
+					<p><strong class="text-light-black fw-semibold fs-12">Starts on</strong>
+						<br> <strong class="fw-semibold fs-14 phase-value">{{startDate}}
+							{{phase['start_zone']}}</strong></p>
+					<p><strong class="text-light-black fw-semibold fs-12">Ends on</strong>
+						<br> <strong class="fw-semibold fs-14 phase-value">{{endDate}}
+							{{phase['end_zone']}}</strong></p>
+					<div [innerHTML]="phase['description']" class="content"></div>
```

Comment:
Should we also add `<p><strong class="text-light-black fw-semibold fs-12">Description</strong>`?

Reply:
@lunayach yes it should but in our current version there is not heading for description so I did exactly same. If you say then I can add that?

Reply:
Okay, anyways, it's not a substantial change. For now, we could have it like in the current version.